services/thermal_mgr/thermal_mgr.c

```diff
@@ -2,6 +2,7 @@
 #include "errors.h"
 #include "lm75bd.h"
 #include "console.h"
+#include "logging/logging.h"
```

Comment:
The way we do things on the subteam is just by including the last part of the include so `#include "logging.h"` is the expected way of doing things